tests/containers/podman_integration.pm

```diff
@@ -19,6 +19,28 @@ use containers::common;
 
 my $test_dir = "/var/tmp";
 my $podman_version = "";
+my $quadlet = "";
+
+sub run_tests {
+    my ($log_file, $skip_tests) = @_;
+
+    my $remote = ($log_file =~ /remote/) ? 1 : 0;
+    my $args = ($log_file =~ /root/) ? "--root" : "--rootless";
```

Comment:
Making assumptions if we're running rootful or rootless based on the log filename can lead to undesired state.

Given that the `$log_file` is predictible, I would suggest to replace the `$log_file` parameter with a `$root` boolean. From there we can then deduce `$log_file` and `$args`.

Reply:
I used `$rootless` instead of `$root` as it seems as a more proper name for a boolean.